### ISSUE TITLE
Notebook default layout

### ISSUE THREAD

**Issue by petetronic** [posit-dev/positron#449]
- Show Primary Side Bar on left and open (if room, otherwise hidden)
- Editor is in the center and shows untitled Notebook 
- Panel is minimized by default in the bottom center (Console is still the default)
- Show Secondary Side Bar with Workspace with Environment showing and Plots is collapsed

**Comment by petetronic** [comment]
Some of this is implemented, but the last requirement of showing the Session and Variables in the secondary side bar (formerly called Workspace and Environment).

**Comment by cindyytong** [comment]
@pete May I close out this issue as a duplicate captured in point 1 in this new ticket #10663

**Comment by pete** [comment]
> [@pete](https://github.com/pete) May I close out this issue as a duplicate captured in point 1 in this new ticket [#10663](https://github.com/posit-dev/positron/issues/10663)

Yes, after examining #10663, it seems to me like closing #10663 would close this one.  I am probably not the best person to ask.
